docs/apache-airflow-providers-oracle/operators/index.rst

```diff
@@ -0,0 +1,87 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+.. _howto/operators:oracle:
+
+Oracle Operators
+================
+The Oracle connection type provides connection to a Oracle database.
+
+Execute SQL in an Oracle database
+---------------------------------
+
+To execute arbitrary SQL in an Oracle database, use the
+:class:`~airflow.providers.oracle.operators.oracle.OracleOperator`.
+
+    .. code-block:: python
+
+        run_sql = OracleOperator(
+            task_id="run_sql",
+            oracle_conn_id="oracle_default",
+            sql="SELECT 1 FROM DUAL",
+            autocommit=True,
+        )
+
+Execute a Stored Procedure in an Oracle database
+------------------------------------------------
+
+To execute a Stored Procedure in an Oracle database, use the
+:class:`~airflow.providers.oracle.operators.oracle.OracleStoredProcedureOperator`.
+
+Assume a stored procedure exists in the database that looks like this:
+
+    .. code-block:: sql
+
+        CREATE OR REPLACE PROCEDURE
+        TEST_PROCEDURE (val_in IN INT, val_out OUT INT) AS
+        BEGIN
+        val_out := val_in * 2;
+        END;
+        /
+
+This stored procedure accepts a single integer argument, val_in, and outputs
+a single integer argument, val_out. This can be represented with the following
+call using :class:`~airflow.providers.oracle.operators.oracle.OracleStoredProcedureOperator`
+with parameters passed positionally as a list:
+
+    .. code-block:: python
+
+        run_stored_procedure = OracleStoredProcedureOperator(
+            task_id="run_stored_procedure",
+            oracle_conn_id="oracle_default",
+            procedure="TEST_PROCEDURE",
+            parameters=[3, int],
+        )
+
+
+Alternatively, parameters can be passed as keyword arguments using a dictionary
+as well.
+
+    .. code-block:: python
+
+        run_stored_procedure = OracleStoredProcedureOperator(
```

Comment:
The examples should be in example dags and be refrenced in the docs (see other peoviders for refrence)
Hard coding examples in the docs is problematic as its very hard to keep it updated when code of the operators changed

Reply:
Agree.